scripts/utils/releaseDocumentation.js

```diff
@@ -16,6 +16,7 @@ function buildDocsForVersion(version) {
     exec.execSync(`npm run docusaurus docs:version ${version}`);
     exec.execSync(`git add .`);
     exec.execSync(`git commit -m "Publish docs version ${version}"`);
+    exec.execSync(`git push origin master"`);
```

Comment:
@asafkorem @igorgn anything to worry about in terms of release script? Is it cool to hardcode master this way?

Reply:
IIUC, technically, the only use cases where this would be invalid is where -
1. We would somehow want to preview docs of a pre-release
2. We would rename master
Both unlikely...
Or have I overlooked something?

Reply:
mmm good point, I think we can take the upstream branch from the `process.env` args..

Reply:
[Done](https://github.com/wix/Detox/pull/3313/commits/1dc4c24521867f215e4feb121e266fcab07e536e).

Reply:
Oh cool. Good to be on the safe side.